tasks/main.yml

```diff
@@ -6,6 +6,19 @@
   become: yes
   file: path={{coreos_kubelet_install_dir}} state=directory mode=0755
 
+- name: Create manifest dir
+  become: yes
+  file: path={{etc_kubernetes}} state=directory mode=0755
+
+- name: Create kubelet config dir
+  become: yes
+  file: path={{kubelet_config_dir}} state=directory mode=0755
+
+- name: Copy kubelet config
+  become: yes
+  template: src={{kubelet_config_file_src}} dest={{kubelet_config_dir}}/config.yaml mode=0755
+  notify: reload systemd
```

Comment:
Copying just the config file doesn't need a daemon-reload; copying kubelet.service does which is taken care below.

Reply:
Yes, I used the interlinked handler. Made change to just restart kubelet on config change.